docs/core_modules/query_modules/chat_engines/root.md

```diff
@@ -24,7 +24,8 @@ To stream response:
 ```python
 chat_engine = index.as_chat_engine()
 streaming_response = chat_engine.stream_chat("Tell me a joke.")
-streaming_response.print_response_stream() 
```

Comment:
curious: why don't we preserve this API?

Reply:
we probably should!